victim/Dockerfile

```diff
@@ -4,12 +4,30 @@ RUN apk update
 RUN apk add tcpdump
 
 ADD requirements.txt .
+
+RUN apk add gcc
+RUN apk add libc-dev
+RUN apk add linux-headers
+
 RUN pip install -r /requirements.txt
 
+RUN pip install uwsgi==2.0.14
+RUN apk add uwsgi-python3
+
+WORKDIR /var/www
 ADD app.py .
 ADD templates templates
+ADD static static
 ADD sniff.py .
+ADD victim.wsgi .
+ADD victim.com.conf /etc/apache2/sites-available/
+
+RUN apk update && apk add apache2-dev
```

Comment:
you can combine all the apk commands into a single one
`RUN apk update && apk add apache2-dev tcpdump [...]`
This is better for caching the image, especially if you do it on line 4 where the first apk command is now.